addons/mail/static/src/discuss/call/common/call_settings.xml

```diff
@@ -72,17 +74,19 @@
                 <t t-if="userSettings.useBlur">
                     <div class="mb-3 d-flex align-items-center flex-wrap">
                         <label class="d-flex align-items-center flex-wrap mw-100 cursor-pointer" title="Background blur intensity" aria-label="Background blur intensity">
-                            <span class="me-2 text-truncate">Background blur intensity <t t-esc="userSettings.backgroundBlurAmount"/></span>
-                            <div class="d-flex w-100">
-                                <input class="flex-grow-2 form-range" type="range" min="0" max="20" step="1" t-att-value="userSettings.backgroundBlurAmount" t-on-change="onChangeBackgroundBlurAmount"/>
+                            <span class="me-2 text-truncate">Background blur intensity</span>
+                            <div class="d-flex w-100 align-items-center">
+                                <input class="flex-grow-2 form-range" type="range" min="0" max="20" step="1" t-att-value="userSettings.backgroundBlurAmount" t-on-input="onChangeBackgroundBlurAmount"/>
+                                <span class="p-1 w-50 text-end"><t t-out="Math.floor(userSettings.backgroundBlurAmount * 5)"/>%</span>
                             </div>
                         </label>
                     </div>
                     <div class="mb-3 d-flex align-items-center flex-wrap">
                         <label class="d-flex align-items-center flex-wrap mw-100 cursor-pointer" title="Edge blur intensity" aria-label="Edge blur intensity">
                             <span class="me-2 text-truncate">Edge blur intensity</span>
-                            <div class="d-flex w-100">
-                                <input class="flex-grow-2 form-range" type="range" min="0" max="20" step="1" t-att-value="userSettings.edgeBlurAmount" t-on-change="onChangeEdgeBlurAmount"/>
+                            <div class="d-flex w-100 align-items-center">
+                                <input class="flex-grow-2 form-range" type="range" min="0" max="20" step="1" t-att-value="userSettings.edgeBlurAmount" t-on-input="onChangeEdgeBlurAmount"/>
+                                <span class="p-1 w-50 text-end"><t t-out="Math.floor(userSettings.edgeBlurAmount * 5)"/>%</span>
```

Comment:
i would suggest to use `with: 6ch` in .scss files. means three digets + `%` and ` character for buffering... 25% width is way too much and it is displaying well in my PR

Reply:
@zel-odoo  yes it felt a bit too big, but not enough to make me want to create a new scss file just for that in a fix, if you have more finely tuned dimensions in your PR for master, i think that it's good

Reply:
OK, then I will have it at my PR